test/performance/hamburg/generate_test_cases.py

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/python3
+
+import csv
+
+fieldnames = ["testCaseId", "description", "departure", "fromLat", "fromLon", "toLat", "toLon", "origin", "destination",
+              "modes", "category"]
+
+locations = [
+    {"name": "Ahrensburg", "coordinates": "53.671926,10.234838"},
+    {"name": "Stade", "coordinates": "53.5960704,9.4775887"},
+    {"name": "Bad Oldesloe", "coordinates": "53.8052678,10.3822381"},
+    {"name": "Henstedt-Ulzburg", "coordinates": "53.7952480,9.9822267"},
+    {"name": "Elmshorn", "coordinates": "53.7542251,9.6592636"},
+    {"name": "Pinneberg", "coordinates": "53.6550342,9.7976864"},
+    {"name": "Buchholz", "coordinates": "53.3243628,9.8749387"},
+    {"name": "HH Hbf", "coordinates": "53.5531993,10.0064364"},
+    {"name": "HH Flughafen", "coordinates": "53.6326466,10.0066778"},
+    {"name": "HH Landungsbrücken (S)", "coordinates": "53.5462334,9.9711435"},
+    {"name": "HH Harburg", "coordinates": "53.4557227,9.9917669"},
+    {"name": "HH Schlump", "coordinates": "53.5675879,9.9700757"},
+    {"name": "HH Kirschenallee", "coordinates": "53.6246134,10.0726482"},
+    {"name": "HH Wasserturm", "coordinates": "53.6196717,10.0500744"},
+    {"name": "HH Finkenwerder", "coordinates": "53.5364482,9.8789390"},
+    {"name": "HH Mümmelmannsberg", "coordinates": "53.5281075,10.1500277"},
+]
+
+failing_cases = [95,96,161,162,181,182,211,212,223,224,241,242,253,254,271,272,283,284,301,302,313,314,331,332,343,344,373,374,403,404,433,434,451,452,]
+
+rows = []
+
+modes = [
+    {
+        "mode": "TRANSIT|WALK",
+        "category": "transit",
+        "window": ""
+    },
+    {
+        "mode": "FLEX_ACCESS|FLEX_EGRESS|TRANSIT",
```

Comment:
Do you have flex in your data set? Flex mean taxi services that pick you up from home or an on demand bus. If you don't you can remove these from the searches.

Reply:
Not yet, I'll remove this for now